source/installguide/management-server/_pkg_repo.rst

```diff
@@ -53,13 +55,33 @@ information.
 
    [cloudstack]
    name=cloudstack
-   baseurl=http://download.cloudstack.org/centos/$releasever/|version|/
+   baseurl=http://download.cloudstack.org/centos$releasever/|version|/
```

Comment:
I've tested this on CentOS 7.9:

_[root@server.... ~]# cat /etc/yum.repos.d/cloudstack-deleteme.repo
[cloudstack]
name=cloudstack
baseurl=http://download.cloudstack.org/centos$releasever/4.16/
enabled=1
gpgcheck=0_


Then I got:

_[root@server1 ~]# yum install cloudstack-management
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
epel/x86_64/metalink                                                                                                                                                                        |  32 kB  00:00:00
 * base: mirrors.netix.net
 * epel: mirrors.netix.net
 * extras: mirrors.netix.net
 * updates: mirrors.netix.net
base                                                                                                                                                                                        | 3.6 kB  00:00:00
http://download.cloudstack.org/centos7/4.16/repodata/repomd.xml: [Errno 14] HTTP Error 404 - Not Found_